controllers/front/AddressController.php

```diff
@@ -84,6 +84,20 @@ public function postProcess()
         }
 
         if (Tools::getValue('delete')) {
+            if ($this->context->cart) {
+                if (
+                    $this->context->cart->id_address_invoice == $id_address
+                    || $this->context->cart->id_address_delivery == $id_address
+                ) {
```

Comment:
Use a single `if statement`, something like : 

```php
if ( Validate::isLoadedObject($this->context->cart)  &&  ( 
      $this->context->cart->id_address_invoice == $id_address
      || $this->context->cart->id_address_delivery == $id_address
)) {
       // Return error message here ...
}
```

Reply:
ok

Reply:
Thanks @dariusakafest, but you forgot something I think.  
`IF (IS_LOADED_OBJECT  && ( IS_INVOICE_ADRESSE || IS_DELIVERY_ADRESSE )){
      // Return error message here 
}`

Reply:
ok